### PR TITLE
Updates to CMSIS-DAP info and minor corrections to V2.2x info.

### PR DESCRIPTION
**Pull request by microbit-carlos**
As I've added some comments https://github.com/microbit-foundation/dev-docs/pull/361/ I've created this PR to add the updates.

### CODE REVIEW

File: hardware/1-5-revision.md [hunk 1]
```diff
@@ -212,16 +212,19 @@ The micro:bit has an on-board USB communications stack, that is built into the f
...
 |    | [Communications Device Class (CDC)](https://en.wikipedia.org/wiki/USB_communications_device_class)
+|    | [CMSIS-DAP HID & WinUSB](https://arm-software.github.io/CMSIS_5/DAP/html/index.html)
+|    | [WebUSB CMSIS-DAP HID](https://wicg.github.io/webusb/)
 | More Info | [DAPLink](../../software/daplink-interface/)
 
 ### Debugging
 
-The interface processor can be used with special host tools to debug code that is running on the application processor. It connects to the application processor via 4 signal wires. The KL26 interface processor code can also be debugged via its integral SWD software debug interface, for example to load initial bootloader code into this processor at manufacturing time, or to recover a lost bootloader.
+The interface processor can be used with special host tools to debug code that is running on the application processor. It connects to the application processor via  2-pin Serial Wire Debug (SWD). The KL26 interface processor code can also be debugged via its integral SWD software debug interface, for example to load initial bootloader code into this processor at manufacturing time, or to recover a lost bootloader.
```
Comment (microbit-robert) [inline hardware/1-5-revision.md:221]:
```suggestion
The interface processor can be used with special host tools to debug code that is running on the application processor. It connects to the application processor via  2-pin Serial Wire Debug (SWD). The KL26 interface processor code can also be debugged via its internal SWD software debug interface, for example to load initial bootloader code into this processor at manufacturing time, or to recover a lost bootloader.
```

**Review by microbit-robert — CHANGES_REQUESTED** [review]
@microbit-carlos One suggested change, which is just one of your corrections not applied in one case. Otherwise, this all looks good. Thanks.

**Comment by microbit-carlos** [comment]
Great, thanks! just committed the patch to the PR as well, so feel free to squash on merge 👍

**Review by microbit-robert — COMMENTED** [review]
LGTM.

**Pull request merged**